spec/filters/scheduling_spec.rb

```diff
@@ -58,7 +58,9 @@
           end
           .then_after(1.2, "wait then translate again") do
             subject.filter(event)
-            wait(0.1).for{event.get("[translation]")}.to eq("12"), "field [translation] did not eq '12'"
+            try(5) do
```

Comment:
these changes are unrelated - they helped with getting CI to :green_circle: more predictably